continuous-async-profiler/src/main/java/com/github/krzysztofslusarski/asyncprofiler/ContinuousAsyncProfilerNotManageablePropertiesRepository.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Krzysztof Slusarski
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.krzysztofslusarski.asyncprofiler;
+
+public interface ContinuousAsyncProfilerNotManageablePropertiesRepository {
+    ContinuousAsyncProfilerNotManageableProperties getAsyncProfilerNotManageableProperties();
```

Comment:
to be consistent with naming in ManagableProperties you could rename it to just `getNotManageableProperties()`